### DISCUSSION TITLE
embla-carousel 8.0 release has been published 4 days ago. We don't need to restrict its version anymore.

### DISCUSSION THREAD

**Discussion by Tomycl** [mantinedev/mantine#5827] [Feedback]
[embla-carousel](https://www.npmjs.com/package/embla-carousel) v8.0.0 is out!
In 2023, our update fixed an unknown error referencing embla-carousel 8.0.0-alpha within the Carousel component. 
Our solution was to restrict the embla-carousel version to 7.1.0. 
<img width="832" alt="image" src="https://github.com/mantinedev/mantine/assets/33180830/f6e9332e-0dad-4a8b-b1fe-4560999652ab">

Now that its version 8.0 has been released, And I've done a quick test and it works fine. 
Therefore, I suggest updating the restriction for yarn/npm installation on the official website [Carousel | Mantine](https://mantine.dev/x/carousel/).
from:
```
yarn install embla-carousel-react@^7.1.0 @mantine/carousel
npm install embla-carousel-react@^7.1.0 @mantine/carousel
```

to:
```
npm install embla-carousel-react @mantine/carousel
npm install embla-carousel-react @mantine/carousel
```

**Comment by Adg1** [comment]
Does not work as expected with embla-carousel 8. For eg, it is not respecting `draggable={false}`

**Comment by max-degterev** [comment]
I think in order to be React 19 compatible this needs to be bumped to actual version, @rtivital would this be possible?

**Reply by rtivital** [reply]
Embla 7 is compatible with react 19.

**Reply by max-degterev** [reply]
<img width="339" alt="Screenshot 2024-12-17 at 12 10 53 PM" src="https://github.com/user-attachments/assets/72832a3c-3026-4407-98a3-d35650b1e818" />

Defined React 18 as latest peer. I mean it doesn't crash sure but it's not specified as compatible

**Reply by rtivital** [reply]
The version of embla package will be updated with 8.0 Mantine release next year. It is a breaking change, it is not planned to be implemented in minor or patch release.

**Reply by michaelrojasGL** [reply]
> The version of embla package will be updated with 8.0 Mantine release next year. It is a breaking change, not planned to be implemented in minor or patch release.

Is there any set date for the 8.0 release? We are planning on updating to React 19 but encountered the same problem with Embla.

**Reply by rtivital** [reply]
See https://alpha.mantine.dev/changelog/8-0-0/
